### PR TITLE
Not Found Page

### PR DESCRIPTION
**Pull request by r-aya**
Created Not Found Page; non-existing URLS redirect here

## Notion ticket link

[404 Page](https://www.notion.so/uwblueprintexecs/404-Page-01a5c7ef96f9452e82a41dbd771fdf34)

## Implementation description
* Used SVGs from the Figma and styles from theme file

## Steps to test
1. Go to a non-existent URL

## What should reviewers focus on?
* Making sure there's no repetitions in background image for different window dimensions
* Page closely resembles Figma design

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have run docker-compose up and my project compiled
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by chriistine — DISMISSED** [review]
Hey I think you have some merge conflicts and yarn lint errors - are you able to fix those first?

**Comment by Tayef-Shah** [comment]
This is the ticket for the not found page right? @r-aya 
https://www.notion.so/uwblueprintexecs/404-Page-01a5c7ef96f9452e82a41dbd771fdf34

File: frontend/yarn.lock [hunk 1]
```diff
@@ -1172,9 +1172,9 @@
     regenerator-runtime "^0.13.4"
 
 "@babel/runtime@^7.3.1", "@babel/runtime@^7.4.4", "@babel/runtime@^7.8.3":
-  version "7.16.3"
```
Comment (susheelk) [inline frontend/yarn.lock:1175]: Hey we decided to not commit yarn.lock files for now. Could you remove this from your PR? Just do `git checkout origin/development yarn.lock`, then commit and push

**Review by susheelk — CHANGES_REQUESTED** [review]

**Review by susheelk — APPROVED** [review]

**Review by susheelk — CHANGES_REQUESTED** [review]
Hey, the background image doesn't seem to fill the whole screen. This is what it looks like for me: 
image

Could you please look into this? Thanks!

**Review by susheelk — APPROVED** [review]
Good job! Looks good to merge :partying_face:

**Pull request merged**